scripts/create_gitlab_snapshot.py

```diff
@@ -53,18 +69,29 @@ def gitlab_volume_info() -> Mapping[str, JSON]:
 
 
 def shutdown_instance(instance: Mapping[str, JSON]):
-    instance_ids = [instance['InstanceId']]
-    log.info('Preparing to stop GitLab instance for %s, waiting 10 seconds '
+    instance_id = instance['InstanceId']
+    log.info('Preparing to stop GitLab instance for %r, waiting 10 seconds '
              'before proceeding. Hit Ctrl-C to abort',
              config.deployment_stage)
     sleep(10)
-    log.info('Stopping instance %s', instance_ids)
-    aws.ec2.stop_instances(InstanceIds=instance_ids)
+    log.info('Stopping instance %r', instance_id)
+    aws.ec2.stop_instances(InstanceIds=[instance_id])
     log.info('Waiting for the GitLab instance to stop')
     waiter = aws.ec2.get_waiter('instance_stopped')
-    waiter.wait(InstanceIds=instance_ids,
+    waiter.wait(InstanceIds=[instance_id],
+                WaiterConfig=dict(MaxAttempts=9999, Delay=15))
+    log.info('Instance %r has stopped', instance_id)
+
+
+def restart_instance(instance: Mapping[str, JSON]):
```

Comment:
```suggestion
def start_instance(instance: Mapping[str, JSON]):
```

I think "to restart" implies stopping, then starting. This  just starts it.